### PR TITLE
EVEREST-238 Added easy installation script

### PR DESCRIPTION
**Pull request by gen1us2k**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+chmod +x everestctl
+echo "Deploying Backends using docker-compose"
+curl -sL  https://raw.githubusercontent.com/percona/percona-everest-backend/main/quickstart.yml -o quickstart.yml
+docker-compose -f quickstart.yml up -d
+
+echo "Using default k8s cluster to provision everest without backups enabled and monitoring"
+echo "You can run ./everestctl for the wizard setup"
+echo ""
+echo "Also you can use --kubeconfig to specify a path for kubeconfig you want to use"
+./everestctl install operators --backup.enable=false --everest.endpoint=http://127.0.0.1:8080  --monitoring.enable=false --operator.mongodb=true --operator.postgresql=true --operator.xtradb-cluster=true --skip-wizard
```
Comment (oksana-grishchenko) [inline install.sh:46]: should we provide `--name` here?

**Review by oksana-grishchenko — APPROVED** [review]

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+if [[ $os == "linux" ]]
+then
+	echo "Your provisioned Everest instance is available at http://127.0.0.1:8080"
+	exit
+fi
```
Comment (michal-kralik) [inline install.sh:52]: Can we print this message for other OS as well?

**Review by michal-kralik — APPROVED** [review]

File: install.sh (hunk 1 above)
Comment (gen1us2k) [reply install.sh:46]: No. CLI handles it byitself

File: install.sh (hunk 2 above)
Comment (gen1us2k) [reply install.sh:52]: We can improve it later but by default it opens everest in browser

**Pull request merged**